mbuild/compound.py

```diff
@@ -3022,37 +3004,24 @@ def save(
         residues : str of list of str
             Labels of residues in the Compound. Residues are assigned by
             checking against Compound.name.
-        combining_rule : str, optional, default='lorentz'
-            Specify the combining rule for nonbonded interactions. Only relevant
-            when the `foyer` package is used to apply a forcefield. Valid
-            options are 'lorentz' and 'geometric', specifying Lorentz-Berthelot
-            and geometric combining rules respectively.
-        foyer_kwargs : dict, optional, default=None
-            Keyword arguments to provide to `foyer.Forcefield.apply`.
-            Depending on the file extension these will be passed to either
-            `write_gsd`, `write_hoomdxml`, `write_lammpsdata`,
-            `write_mcf`, or `parmed.Structure.save`.
-            See `parmed structure documentation
-            <https://parmed.github.io/ParmEd/html/structobj/parmed.structure.Structure.html#parmed.structure.Structure.save>`_
-        parmed_kwargs : dict, optional, default=None
-            Keyword arguments to provide to :meth:`mbuild.Compound.to_parmed`
+            #TODO 1.0: Update this kwargs, pass link to GMSO
         **kwargs
             Depending on the file extension these will be passed to either
-            `write_gsd`, `write_hoomdxml`, `write_lammpsdata`, `write_mcf`, or
+            `write_gsd`, `write_lammpsdata`, `write_mcf`, or
```

Comment:
Would it be better to pass the following conversion mapping in the documentation. Then we just link to https://github.com/mosdef-hub/mbuild/blob/e42cd0166b7ea1647dba6f89c2e6f0faf51528c6/mbuild/conversion.py#L936 for more details?
```python
`{GMSO: {".gro", ".gsd". ".data", ".xyz", ".mcf", ".top"},
ParmEd: {".mol2", ".pdb", ".prmtop", ".cif", ".crd"},
PyBel:{".sdf"}}`
```

Reply:
Good idea. This should be done, along with the other suggestions.